File: linguagens/Java/exercicios/math.java
import java.util.Scanner;
import java.lang.Math;

public class math
{
	public static void main(String[] args) {

		double base, altura, diagonal, perimetro, area;

		Scanner sc = new Scanner(System.in);

		System.out.println("coloque a Base do retangulo: ");

		base = sc.nextDouble();

		System.out.println("coloque a altura do retangulo: ");

		altura = sc.nextDouble();


		area = (altura * base);

		System.out.println("area do retangulo: " + area);

		perimetro = 2 * (base + altura);

		System.out.println("perimetro do retangulo: " + perimetro);

		diagonal = Math.sqrt((base*base) + (altura*altura));

		System.out.println("diagonal do retangulo: " + String.format("%.4f",diagonal));
		
		sc.close();
	}
}

File: linguagens/C#/Exercicios/math.cs
using System;
using System.Globalization;

namespace Programa {
class Program {
	static void Main(string[] args) {


		CultureInfo CI = CultureInfo.InvariantCulture;

		double altura, base1, area, perimetro, diagonal;

		Console.WriteLine("Coloque a base: ") ;
		base1 = double.Parse(Console.ReadLine(), CI);
		
		Console.WriteLine("Coloque a altura: ") ;
		altura = double.Parse(Console.ReadLine(), CI);
		
		
		area = base1 * altura; 
		
		perimetro = 2 * (base1 +  altura); 
		
		diagonal = Math.Sqrt((altura * altura) + (base1 * base1)); 
		
		
		Console.WriteLine("area = " + area.ToString("F2", CI)); 
		Console.WriteLine("perimetro = " + perimetro.ToString("F2", CI)); 
		Console.WriteLine("diagonal = " + diagonal.ToString("F2", CI)); 
		
	}
}
}